### PR TITLE
Remove shielding advice logic from front end

### PR DESCRIPTION
**Pull request by sanisc**
Shielding logic has all been moved to a backend store proc call so any front code can now be removed

### CODE REVIEW

File: vulnerable_people_form/__init__.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 from vulnerable_people_form.form_pages.shared.querystring_utils import append_querystring_params
 from . import form_pages
 from .form_pages.shared.form_utils import postcode_with_spaces
-from .integrations import nhs_openconnect_id, persistence, ladcode_shielding_advice_lookup
+from .integrations import nhs_openconnect_id, persistence
 
 _ENV_DEVELOPMENT = "DEVELOPMENT"
 _DEFAULT_SHIELDING_ADVICE_FILE_PREFIX = "vulnerable_people_form/integrations/data/local-area-shielding-advice-"
```
Comment (jk132) [inline vulnerable_people_form/__init__.py:18]: This is no longer needed?

**Review by jk132 — CHANGES_REQUESTED** [review]
Looks very reasonable in terms of stripping out the old code (aside from the one comment).
Assuming that the new implementation (with appropriate tests) is included elsewhere.

File: vulnerable_people_form/__init__.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (sanisc) [reply vulnerable_people_form/__init__.py]: Yes that has been removed now

**Comment by sanisc** [comment]
> Looks very reasonable in terms of stripping out the old code (aside from the one comment).
> Assuming that the new implementation (with appropriate tests) is included elsewhere.

Correct the implementation is already in production - this was just the left over code that needed removing

**Comment by whpearson** [comment]
I think we can cleanup the gatling tests, we have data for non tiering logic svp_submission_data.json and also we should remove reference to the TIERING logic flag from the load tests https://github.com/alphagov/govuk-shielded-vulnerable-people-service/blob/master/loadtests/src/test/scala/svp/ShieldedVulnerablePeople.scala#L10

**Comment by whpearson** [comment]
I think I left some reference to la_shielding in the tests that can be removed as well. 

E.g.  in tests/form_pages/shared/test_location_tier.py there is test_update_location_status_by_postcode_should_update_session_when_tiering_logic_enabled which has   _current_app.shielding_advice.advice_from_la_shielding = lambda ladcode: shielding_advice

**Comment by sanisc** [comment]
> I think I left some reference to la_shielding in the tests that can be removed as well.
> 
> E.g. in tests/form_pages/shared/test_location_tier.py there is test_update_location_status_by_postcode_should_update_session_when_tiering_logic_enabled which has _current_app.shielding_advice.advice_from_la_shielding = lambda ladcode: shielding_advice

These have been removed.

**Review by whpearson — APPROVED** [review]
Tested locally, automatically and manually. Changes look good.

**Review by jk132 — APPROVED** [review]
LGTM

**Pull request merged**